apps/etterlatte-saksbehandling-ui/client/src/components/aktivitetsplikt/brev/LoependeUnntakInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import { Alert } from '@navikt/ds-react'
+import React from 'react'
+import { useAktivitetspliktOppgaveVurdering } from '~components/aktivitetsplikt/OppgaveVurderingRoute'
+import { IAktivitetspliktVurderingNyDto } from '~shared/types/Aktivitetsplikt'
+
+function loeperUnntakEtterSisteVurdering(vurdering: IAktivitetspliktVurderingNyDto): boolean {
```

Comment:
Vi kaller vel "vurdering" her både for aktivitet og unntak, så kanskje `loeperUnntakEtterSisteAktivitet`?